tds/src/main/java/thredds/server/wfs/WFSGetFeatureWriter.java

```diff
@@ -14,16 +17,29 @@
 	
 	private PrintWriter response;
 	private String fileOutput;
+	private final String namespace;
+	private final String server;
 	
+	private String writeGeometryInformation(Point point) {
+		return "<gml:Point srsName=\"http://www.opengis.net/gml/srs/epsg.xml@900913\" srsDimension=\"2\">"
+				+ "<gml:pos>50.0 50.0</gml:pos>"
+				+ "</gml:Point>";
+	}
 	
 	/**
-	 * Initiate the response with an XML file with an XML header and the FeatureCollection tag. Write bounding box information.
+	 * Writes headers and bounding box
 	 */
-	public void startXML() {
-		fileOutput += "<?xml version=\"1.0\" encoding=\"UTF-8\"?>";
-		fileOutput += "<wfs:FeatureCollection xsi:schemaLocation=\"http://www.opengis.net/wfs/2.0 http://schemas.opengis.net/wfs/2.0/wfs.xsd\" xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\""
-				+ " xmlns:xlink=\"http://www.w3.org/1999/xlink\" xmlns:gml=\"http://opengis.net/gml/3.2\" xmlns:fes=\"http://www.opengis.net/fes/2.0\" xmlns:ogc=\"http://www.opengis.net/ogc\""
-				+ " xmlns:wfs=\"http://opengis.net/wfs/2.0\" xmlns:tds=\"http://localhost:8080/thredds/wfs/results/tds\" xmlns=\"http://www.opengis.net/wfs/2.0\" version=\"2.0.0\" numberMatched=\"1\" numberReturned=\"1\">"
+	private void writeHeadersAndBB() {
+		fileOutput += "<wfs:FeatureCollection xsi:schemaLocation=" + WFSXMLGeneral.encQuotes("http://www.opengis.net/wfs/2.0 http://schemas.opengis.net/wfs/2.0/wfs.xsd " + namespace + " " + server + "?request=DescribeFeature" + WFSXMLGeneral.AMPERSAND + "service=wfs" + WFSXMLGeneral.AMPERSAND + "version=2.0.0")

```

Comment:
Good use of variables and utilities in building the schemaLocation here. This schemaLocation should contain a valid `DescribeFeatureType` URL. Could you please revise the URL?

http://cite.opengeospatial.org/pub/cite/files/edu/wfs/text/operations.html#describefeaturetype

Reply:
Yeah, corrected to DescribeFeatureType and typename is set to TDSNAMESPACE:hru_soil_moist